lib/cloud_controller/runner.rb

```diff
@@ -39,6 +39,9 @@ def initialize(argv)
       app     = builder.build(@config, request_metrics, request_logs)
 
       @server = if @config.get(:webserver) == 'puma'
+                  max_connections_per_process = @config.get(:puma, :max_connections)
```

Comment:
I think the correct place for this is the `setup_db` method. There `@config.get(:db)` is passed to the `DB.load_models` method - this config hash should be adapted according to the new configuration option.